pipelines/RunIsrForCrosstalkSources.yaml

```diff
@@ -0,0 +1,22 @@
+description: Prepare overscanSources (for ISR/interchip crosstalk).
```

Comment:
Please elaborate in description (here and for the other one) when each pipeline should be used, since I am confused when to use each one versus the existing pipeline `RunIsr.yaml`. That one describes itself as `ip_isr handling of DECam-specific inter-chip crosstalk.  See DM-25348.` 🤯

Reply:
I've updated the descriptions to be clearer.  `RunIsr.yaml` correctly handles only intra-chip crosstalk (despite what I wrote in the previous description).  `RunIsrForCrosstalkSources.yaml` pre-runs overscan to generate the crosstalk sources, which can then be used with the `RunIsrWithCrosstalk.yaml` to do the full intra- and inter-chip crosstalk corrections.